common/test/acceptance/pages/studio/users.py

```diff
@@ -8,19 +8,6 @@
 from . import BASE_URL
 
 
-def wait_for_ajax_or_reload(browser):
```

Comment:
@dan-f This method was making a test flaky, I removed it after discussing with @jzoldak, see our above conversation.